rust/agama-cli/src/config.rs

```diff
@@ -99,6 +99,7 @@ pub async fn run(
     http_client: BaseHTTPClient,
     monitor: MonitorClient,
     subcommand: ConfigCommands,
+    insecure: bool,
```

Comment:
Why not pass the GlobalOpts? If tomorrow we add a, let's say, `--verbose` argument, you might need to add another argument.